engine/platform/win32/sys_win.c

```diff
@@ -46,6 +62,92 @@ qboolean Sys_DebuggerPresent( void )
 	return IsDebuggerPresent();
 }
 
+static void Sys_SetTimerHighResolution( void )
+{	
+	HMODULE hndl;
+	pfnNtSetTimerResolution_t pNtSetTimerResolution;
+	pfnNtQueryTimerResolution_t pNtQueryTimerResolution;
+	
+	hndl = GetModuleHandle( "ntdll.dll" );
+	pNtSetTimerResolution = (pfnNtSetTimerResolution_t)GetProcAddress( hndl, "NtSetTimerResolution" );
+	pNtQueryTimerResolution = (pfnNtQueryTimerResolution_t)GetProcAddress( hndl, "NtQueryTimerResolution" );
+
+	if( pNtSetTimerResolution && pNtQueryTimerResolution )
+	{
+		// undocumented NT API functions, for some mystery reason works better than timeBeginPeriod
+		ULONG min, max, cur;
```

Comment:
Wrong format.

Reply:
What do you mean?

Reply:
Add spaces between brackets and other characters